Assets/MixedRealityToolkit/_Core/Definitions/Devices/ControllerMappingLibrary.cs

```diff
@@ -17,213 +17,161 @@ public static class ControllerMappingLibrary
+
+        /// <summary>
+        /// INDEXFINGER_LEFT_CONTROLLER
+        /// </summary>
+        public const string MIXEDREALITY_OPENVR_AXIS20 = "MIXEDREALITY_OPENVR_AXIS20";
+
+        /// <summary>
+        /// INDEXFINGER_RIGHT_CONTROLLER
+        /// </summary>
+        public const string MIXEDREALITY_OPENVR_AXIS21 = "MIXEDREALITY_OPENVR_AXIS21";
+
+        /// <summary>
+        /// MIDDLEFINGER_LEFT_CONTROLLER
+        /// </summary>
+        public const string MIXEDREALITY_OPENVR_AXIS22 = "MIXEDREALITY_OPENVR_AXIS22";
+
+        /// <summary>
+        /// MIDDLEFINGER_RIGHT_CONTROLLER
+        /// </summary>
+        public const string MIXEDREALITY_OPENVR_AXIS23 = "MIXEDREALITY_OPENVR_AXIS23";
+
+        /// <summary>
+        /// RINGFINGER_LEFT_CONTROLLER
+        /// </summary>
+        public const string MIXEDREALITY_OPENVR_AXIS24 = "MIXEDREALITY_OPENVR_AXIS24";
+
+        /// <summary>
+        /// RINGFINGER_RIGHT_CONTROLLER
+        /// </summary>
+        public const string MIXEDREALITY_OPENVR_AXIS25 = "MIXEDREALITY_OPENVR_AXIS25";
+
+        /// <summary>
+        /// PINKYFINGER_LEFT_CONTROLLER
+        /// </summary>
+        public const string MIXEDREALITY_OPENVR_AXIS26 = "MIXEDREALITY_OPENVR_AXIS26";
+
+        /// <summary>
+        /// PINKYFINGER_RIGHT_CONTROLLER
+        /// </summary>
+        public const string MIXEDREALITY_OPENVR_AXIS27 = "MIXEDREALITY_OPENVR_AXIS27";
+
+        #endregion Constants
+
+        #region InputAxisConfig
 
         /// <summary>
-        /// Get the Input Manager string Mappings for a specific controller type
+        /// Get the InputManagerAxis data needed to configure the Input Mappings for a controller
         /// </summary>
         /// <param name="type">The type of controller to retrieve configuration for</param>
         /// <returns></returns>
-        public static string[] GetInputManagerMappings(string type)
+        public static InputManagerAxis[] GetOpenVRInputManagerAxes
         {
-            return InputManagerMappings.ContainsKey(type) ? InputManagerMappings[type] : default(string[]);
+            get
+            {
+                return new InputManagerAxis[]
+                {
+                    new InputManagerAxis { Name = MIXEDREALITY_OPENVR_AXIS1, Dead = 0.1f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 1 },
+                    new InputManagerAxis { Name = MIXEDREALITY_OPENVR_AXIS2, Dead = 0.1f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 2 },
+                    new InputManagerAxis { Name = MIXEDREALITY_OPENVR_AXIS4, Dead = 0.1f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 4 },
+                    new InputManagerAxis { Name = MIXEDREALITY_OPENVR_AXIS5, Dead = 0.1f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 5 },
```

Comment:
Why are these 0.1 but the others 0.001?

Reply:
Thumbsticks / Touchpads are 0.1.
Triggers and such need to be a little more precise

Reply:
Shouldn't we keep them all the same, then add a sensitivity modifier after we read the value?